src/main/pages/che-7/contributor-guide/ref_language-server-protocol.adoc

```diff
@@ -1,23 +1,17 @@
+---
+title: Language Server Protocol reference
+keywords: 
+tags: []
+sidebar: che_7_docs
+permalink: che-7/language-server-protocol-reference/
+folder: che-7/contributor-guide
+summary: 
+---
+
 [id="language-server-protocol_{context}"]
 = Language Server Protocol
 
-This paragraph is the reference module introduction and is only optional. Include it to provide a short overview of the module.
-
-A reference module provide data that users might want to look up, but do not need to remember. It has a very strict structure, often in the form of a list or a table. A well-organized reference module enables users to scan it quickly to find the details they want. AsciiDoc markup to consider for reference data:
-
-
-.Unordered lists
-* For more details on writing reference modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
-* Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
-
-.Labeled lists
-Term 1:: Definition
-Term 2:: Definition
+The Language Server Protocol (LSP) defines the protocol used between an editor or IDE and a language server that provides language features like auto complete, go to definition, find all references etc. 
 
-.Tables
-[options="header"]
-|====
-|Column 1|Column 2|Column 3
-|Row 1, column 1|Row 1, column 2|Row 1, column 3
-|Row 2, column 1|Row 2, column 2|Row 2, column 3
-|====
+For more details refer to the link:https://microsoft.github.io/language-server-protocol/[
```

Comment:
```suggestion
For more details, see the link:https://microsoft.github.io/language-server-protocol/[Language Server Protocol website].
```
* Don't use 'refer to', see *IBM Style Guide* pg. 357.
(Or see https://hackmd.io/gmcTtq_gQYe1dyaC0KJ9Ag)